### PR TITLE
boot_800862E0 OK

### PR DESCRIPTION
**Pull request by kyleburnette**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

Pretty much a copy/paste job from OoT.

### CODE REVIEW

File: baserom.mm.us.rev1.z64:Zone.Identifier [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+ZoneId=3
+ReferrerUrl=E:\Legend of Zelda, The - Majora's Mask (U) [!].zip
```
Comment (engineer124) [inline baserom.mm.us.rev1.z64:Zone.Identifier:3]: What's this for?

Comment (jpburnett) [reply baserom.mm.us.rev1.z64:Zone.Identifier:3]: I think it's a remnant from the windows file system from copying something from one location to another. Probably should just be deleted.

Comment (EllipticEllipsis) [reply baserom.mm.us.rev1.z64:Zone.Identifier:3]: Yeah, that always happens when you copy something over from Windows to WSL. We could consider gitignoring them, but it's better really to just make sure to check for and delete them whenever you copy stuff over.

Comment (hensldm) [reply baserom.mm.us.rev1.z64:Zone.Identifier:3]: Looks like this file didn't get removed properly.

File: src/code/z_skelanime.c [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #include "prevent_bss_reordering.h"
 #include "global.h"
+#include "prevent_bss_reordering.h"
```
Comment (hensldm) [inline src/code/z_skelanime.c:3]:
```suggestion
#include "global.h"
```
Does this work by chance?

**Review by hensldm — CHANGES_REQUESTED** [review]
Mainly the  Zone.Identifier file still needs to be deleted.

**Review by hensldm — APPROVED** [review]

**Review by AngheloAlf — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]
Friendly reminder that boot files should be more documented.

**Review by Kenix3 — CHANGES_REQUESTED** [review]
Friendly reminder that boot files need more complete documentation. Let us know if there is too much missing information to do any documentation. Contact me on discord if you need help.

**Comment by EllipticEllipsis** [comment]
Superseded by #769 .

**Pull request closed without merging**